lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -258,7 +260,7 @@ export class EvaluationClientWrapper {
     return presentationSubmission;
   }
 
-  private partitionCheckResults(marked: HandlerCheckResult[]): Map<string, string[]> {
+  private static partitionCheckResults(marked: HandlerCheckResult[]): Map<string, string[]> {
```

Comment:
Is that supposed to be static?

Reply:
Recommended by the IDE. And as per books it is going to be fine. When needed it can be converted to non-static.

Reply:
Is it what we want? I don't think so, otherwise we would have adopted this style already. In my opinion these changes need to be reverted.

Reply:
Fixed